File: src/main/java/com/est/curdsample/dto/TaskPageDto.java
package com.est.curdsample.dto;

import java.util.List;
import lombok.Builder;
import lombok.Data;

public record TaskPageDto(boolean hasNext, List<TaskDto> data) {
}

//@Data
//@Builder
//public class TaskPageDto {
//
//    private boolean hasNext;
//    private List<TaskDto> data;
//
//}

File: src/main/java/com/est/curdsample/dto/TaskPageDto.kt
package com.est.curdsample.dto

data class TaskPageDto(
    val hasNext: Boolean,
    val data: List<TaskDto>,
)


//@Data
//@Builder
//public class TaskPageDto {
//
//    private boolean hasNext;
//    private List<TaskDto> data;
//
//}

